wofs/virtualproduct.py

```diff
@@ -59,6 +68,30 @@ def _load_dsm(self, gbox):
         return xr.Dataset(data_vars={'elevation': (('y', 'x'), dsm)}, coords=_to_xrds_coords(gbox),
                           attrs={'crs': gbox.crs})
 
+def scale_usgs_collection2(data):
+    return data.apply(scale_and_clip_dataarray, keep_attrs=True,
+                      scale_factor=0.275, add_offset=-2000, clip_range=(0, 10000))
```

Comment:
Might be nice to have a comment that references where we can check these values.